packages/dm-core-plugins/src/list/ListPlugin.tsx

```diff
@@ -93,17 +93,24 @@ export const ListPlugin = (props: IUIPlugin & { config?: TListConfig }) => {
     updateItem,
   } = useList<TGenericObject>(idReference, internalConfig.resolveReferences)
 
-  const defaultPaginationRowsPerPage =
-    internalConfig.defaultPaginationRowsPerPage ?? 10
-  const showPagination = useMemo(
-    () => items.length > defaultPaginationRowsPerPage,
-    [items]
-  )
+  const defaultPaginationRowsPerPage = useMemo(() => {
+    let numRows = internalConfig.defaultPaginationRowsPerPage ?? 5
+    numRows = Math.round(numRows)
```

Comment:
I know we joked about it in the meeting, but is it maybe not better to expect user to input a default that makes sense than do 2 operations like this when it might not be necessary? 😅